canonical-websites/docs.ubuntu.com
# Add new content to index page

## Done

- Update of layout and content

There may be some small changes required such as icons.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://localhost:8007>

## Issue / Card

<https://docs.google.com/document/d/1wc8f4Vh8q5-EaYCGNHCjNJmRrY21fFx8YyNZV8DmPUs/edit>

## Screenshots
![2018-03-09-09-02-localhost_8007](https://user-images.githubusercontent.com/501889/37196897-d3eb0fec-2378-11e8-978a-c8d5db636a07.png)
